app/namespace.go

```diff
@@ -574,6 +574,33 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
 						},
 					},
 				},
+				{
+					Name:    "retention",
+					Usage:   "The length of time a closed workflow will be preserved before deletion",
```

Comment:
[Nit] re-word usage a bit as this is a command, not a flag (e.g. "Manages the length of time a closed workflow...")

Reply:
done